### PR TITLE
Feature/xivy 12667 optimization

### PR DESCRIPTION
**Pull request by ivy-fhe**
Implemented feedback:
- Removed dropin installer
- Added maven build step to delete zip-archive
- Added simple process to generate Key-Pair
- Updated README.md to reflect new process

### CODE REVIEW

File: threema-connector/pom.xml [hunk 1]
```diff
@@ -52,6 +52,18 @@
...
+                	<phase>validate</phase>
+                	<goals>
+                   		<goal>run</goal>
+                	</goals>
+	                <configuration>
+	                    <target>
+	                        <delete file="${project.basedir}/lib/threema-msgapi-tool.zip"/>
+	                    </target>
+	                </configuration>
+            	</execution>
```
Comment (ivy-rew) [inline threema-connector/pom.xml]: code is good; but the executions seem a bit overdone in here and therefore harder to maintain.
I'd rather keep all 3 executions in one block ... it's all just about providing the threema.jar which isn't available as normal maven artifact.

File: threema-connector-product/README.md [hunk 2]
```diff
@@ -13,8 +13,9 @@ Credentials and credits are required to send messages. The credentials can be cr
 ## Setup
 This Connector requires an "End-to-End Threema ID". [Request new ID](https://gateway.threema.ch/en/id-request)
 <br> 
-For generating the keys and requesting a new Threema.Gateway ID refer to [Generate keys](https://gateway.threema.ch/en/developer/howto/create-keys/php).
+To request a new Threema.Gateway ID refer to [Generate keys](https://gateway.threema.ch/en/developer/howto/create-keys/php).
 
+For generating the keys you can use the "createKeyPair" process.
 To use the Threema Connector, add the following variables to your Axon Ivy Project:
```
Comment (ivy-rew) [inline threema-connector-product/README.md]: maybe a numbered list of steps to do could slightly simplify the instuctions?

**Review by ivy-rew — APPROVED** [review]

**Comment by ivy-rew** [comment]
some minor comments; you are free to choose whether you want to apply anyone of em.

can't wait to get my threema credits to actually run this stuff :dancers:

File: threema-connector/pom.xml (hunk 1 above)
Comment (ivy-fhe) [reply threema-connector/pom.xml]: Combined all steps into 1

File: threema-connector-product/README.md (hunk 2 above)
Comment (ivy-fhe) [reply threema-connector-product/README.md]: Added step by step instructions

**Review by ivy-rew — APPROVED** [review]
:100:

**Pull request merged**